style.css

```
@@ -16,17 +16,45 @@ html {
     background-color: var(--background);
 }
 
+header.shadow {
+    transition: box-shadow 0.1s ease;
+    box-shadow: 0px 0px 10px 5px var(--main-heading-color);
+}
+
 header {
     position: sticky;
     top: 0;
     background-color: rgba(40, 40, 40, 0.7);
-    backdrop-filter: blur(10px);
+    backdrop-filter: blur(5px);
     padding: 20px;
     z-index: 1000;
     padding-top: 1px;
     border-bottom: solid 1px var(--text-color);
 }
 
+header button {
+    background-color: rgba(40, 40, 40, 0.7);
+    color: var(--accent-color);
+    border-color: var(--accent-color);
+    width: 25%;
+    backdrop-filter: blur(0px);
+    padding-top: 1%;
+    padding-bottom: 1%;
+    /* padding-left: %; */
+    margin-left: 6%;
+    border-radius: 5px;
+    transition: all 0.3s ease;
+}
+
+header button:hover, header button:focus{
+    opacity: 1;
+    background-color:#e54b4b;
+    /* border-color: black; */
+    color: black;
+    border-radius: 15px;
+    box-shadow: 0px 0px 10px var(--accent-color);
+}
+
 #contribution_link a {
```

So that the link will be styled correctly:

```suggestion
#contribution_link {
```